forge-gui/res/cardsfolder/c/case_of_the_locked_hothouse.txt

```diff
@@ -5,6 +5,7 @@ S:Mode$ Continuous | Affected$ You | AdjustLandPlays$ 1 | Description$ You may p
 T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | CheckSVar$ X | SVarCompare$ GE7 | IsPresent$ Card.Self+IsUnsolved | TriggerZones$ Battlefield | Execute$ TrigSolve | TriggerDescription$ To solve — You control seven or more lands. (If unsolved, solve at the beginning of your end step.)
 SVar:TrigSolve:DB$ AlterAttribute | Defined$ Self | Attributes$ Solved
 SVar:X:Count$Valid Land.YouCtrl
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ Player | Description$ Play with the top card of your library revealed.
```

Comment:
yea looks like we need to split it, but not for all players and without extra Description